libs/blocks/global-navigation/global-navigation.css

```diff
@@ -346,23 +478,48 @@ header .gnav-navitem.has-menu > a:after {
   margin-right: 3px;
 }
 
-.gnav-cta {
+.last-link-blue ul li:last-of-type a {
+  color: #1473E6;
+}
+
+/* END MAINNAV */
+
+/* PROFILE */
+
+.feds-profile {
   display: flex;
   align-items: center;
-  padding: 20px 12px;
+  font-size: 14px;
+  position: relative;
+  justify-content: end;
 }
 
-.gnav-cta .con-button {
-  line-height: 36px;
-  border-radius: 20px;
-  padding: 0 14px;
+/* TODO temporarily more specificity until we improve the CSS */
+header .gnav-navitem .feds-signin {
```

Comment:
Does this CSS need to be in the `global-navigations.css` if we have `signIn.css` ?

Reply:
Nope, this is likely a rebase conflict omission. I'll remove it from here.